instrument-repl/src/repl.rs

```diff
@@ -552,15 +552,6 @@ impl Repl {
                     };
                     let json_file = PathBuf::from(file.clone());
 
-                    if !json_file.is_file() {
```

Comment:
this check is required, 
when user type ".nodes <file path>" and 
path is wrong then it protect terminal to get crash

in below ideal case there is no use of this check
- when user right click and select option to fetch the node details
- when path is correct but file is not there then it will create one,

Reply:
Maybe we just add a check for the existence of the parent directory, then. I just want to make sure we don't also show an error if that file happens to not exist. 

I'm debating whether we should just create the directory if it doesn't exist, too.

Reply:
checking the existence for parent directory looks good for me

Creating, directory looks complicated,

Reply:
I believe I've addressed this in 0a3c6fe

Reply:
Per our conversation, I have made updating the config file infallible in f145265